docs/guides/developer/docs/architecture/api/playlists-api.md

```diff
@@ -0,0 +1,215 @@
+[TOC]
+
+# Information
+
+The Playlists API is available since API version 1.11.0.
+
+### GET /api/playlists/playlist.json
+
+Returns a playlist.
+
+__Response__
+
+`200 (OK)`: A playlist as JSON.
+
+| Field              | Type                        | Description                                               |
+|--------------------|-----------------------------|-----------------------------------------------------------|
+| `id`               | [`long`](types.md#basic)    | The playlist identifier                                   |
```

Comment:
This still needs to be adjusted after your last commit (string IDs). And other parts of these docs as well.